stl/inc/chrono

```diff
@@ -5901,8 +5921,8 @@ struct _Chrono_formatter {
             return true;
         case 'H':
             if constexpr (_Is_specialization_v<_Ty, _CHRONO hh_mm_ss>) {
-                if (-_CHRONO hours{24} >= _Val.hours() || _Val.hours() >= _CHRONO hours{24}) {
-                    _THROW(format_error("Cannot localize hh_mm_ss longer than 24 hours."));
+                if (_Val.hours() <= -_CHRONO hours{24} || _CHRONO hours{24} <= _Val.hours()) {

```

Comment:
```suggestion
                if (_Val.hours() >= _CHRONO hours{24}) {
```
`hh_mm_ss:hours()` is the integral number of hours of the duration's absolute value.

Reply:
Great catch, added to next batch of changes.